vb-fe/src/components/select/SelectOption.tsx

```diff
@@ -0,0 +1,47 @@
+/* Copyright (C) 2021 Vibey - All Rights Reserved */
+
+import React, { FunctionComponent } from "react";
+import { components } from "react-select";
+
+import { Badge, Box } from "@chakra-ui/react";
+
+import "components/select/Select.scss";
+
+/**
+ * Custom option for tags to display the label and score
+ *
+ * Need to figure out the correct type for the prop to get rid of any
+ *
+ * @param props options props passed by react-select f
+ */
+export const SelectOption: FunctionComponent<any> = (props) => {
```

Comment:
It looks like we could just add a tag type here for props.  I think we can spread the rest to the option element also, but am not entirely certain about this..
```
type Props = {
  label: string;
  score: number; // I'm not sure if this is a number or string in the FE, actually
}

export const SelectOption = ({label, score, ...rest}: Props) => {
<components.Option {...rest}>
  <Box>
    <span className="text-grey-1>#</span> <span>{label}</span>
    <Badge colorScheme="primary" mx={2}>
      {score}
    </Badge>
  </Box>
</components.Option>
}
```

Reply:
so since this is react-select you cant assign custom props to it. you'll have to derive it from one of the props defined by react select - which I couldnt figure out